### ISSUE TITLE
Extension caches header authorization token

### ISSUE THREAD

**Issue by joshuabromero** [magespecialist/m2-MSP_APIEnhancer#6]
I'm working on a site that uses this extension and noticed a pretty significant issue. When using token authorization, the extension caches the authorization token. Here's what I've done to replicate the issue.

Install extension
Get an admin auth token by using the `/V1/integration/admin/token` endpoint
Copy auth token and set it in the header using `Authorization: Bearer {token}` along with `Content-Type: application/json`
Hit a default Magento 2 API endpoint: `/V1/categories/2`
Disable the `Authorization` header and keep the `Content-Type` header
In my tests, the request comes back successful.

**Comment by magedevjosh** [comment]
Looks like it may be related to the specific site I'm working on and not an actual issue with the extension.

**Comment by joshuabromero** [comment]
Nope, still an issue. Turns out when I installed the module, the cache was disabled by default. When I enabled it on the fresh copy of 2.3.0, the same problem occurred.

**Comment by hawksakr** [comment]
we had to go through the same issue on Magento 2.3.2
